base/bucket_gocb.go

```diff
@@ -914,20 +914,35 @@ func (bucket CouchbaseBucketGoCB) GetWithXattr(k string, xattrKey string, rv int
 
 		switch err {
 		case nil:
-			// Successfully retrieved doc and xattr.  Copy the contents into rv, xv and return.
+			// Successfully retrieved doc and (optionally) xattr.  Copy the contents into rv, xv and return.
 			err = res.Content("", rv)
 			if err != nil {
-				LogTo("gocb", "Unable to retrieve xattr content for key=%s, xattrKey=%s: %v", k, xattrKey, err)
+				LogTo("gocb", "Unable to retrieve document content for key=%s, xattrKey=%s: %v", k, xattrKey, err)
 				return false, err, nil
 			}
-			err = res.Content(xattrKey, xv)
-			if err != nil {
-				LogTo("gocb", "Unable to retrieve xattr content for key=%s, xattrKey=%s: %v", k, xattrKey, err)
-				return false, err, nil
+			if err != gocbcore.ErrSubDocBadMulti {
```

Comment:
This was residual from before ErrSubDocBadMulti was split into a different case.  You're correct - right now the err check doesn't do anything, and it always executes this case (err is always nil at this point).  Will clean up.